builds/misc/images-release.yaml

```diff
@@ -0,0 +1,602 @@
+          FolderPath: $(Build.BinariesDirectory)/publish
+          Pattern: Microsoft.Azure.WebJobs.Extensions*.nupkg
+          inlineOperation: |
+              [
+                {
+                    "keyCode": "CP-401405",
+                    "operationSetCode": "NuGetSign",
+                    "parameters": [],
+                    "toolName": "sign",
+                    "toolVersion": "1.0"
+                },
+                {
+                    "keyCode": "CP-401405",
+                    "operationSetCode": "NuGetVerify",
+                    "parameters": [],
+                    "toolName": "sign",
+                    "toolVersion": "1.0"
+                }
+              ]
+          signConfigType: inlineSignParams
+      - template: templates/image-linux.yaml
+        parameters: 
+          imageName: azureiotedge-agent
+          name: "Edge Agent"
+          project: Microsoft.Azure.Devices.Edge.Agent.Service
+      - template: templates/image-linux.yaml
+        parameters: 
+          imageName: azureiotedge-hub
+          name: "Edge Hub"
+          project: Microsoft.Azure.Devices.Edge.Hub.Service
+      - template: templates/image-linux.yaml
+        parameters: 
+          imageName: azureiotedge-simulated-temperature-sensor
+          name: "Temperature Sensor"
+          project: SimulatedTemperatureSensor
+      - task: Bash@3
+        displayName: "Build iotedge-diagnostics"
+        inputs: 
+          arguments: "-i azureiotedge-diagnostics -n microsoft -P iotedge-diagnostics -c $(configuration)"
+          filePath: ./scripts/linux/buildEdgelet.sh
+          targetType: filePath
+      - bash: |
+            scripts/linux/buildEdgelet.sh -i azureiotedge-diagnostics -n microsoft -P iotedge-diagnostics -c $(configuration) &&
+            scripts/linux/buildImage.sh -r $(registry.address) -u $(registry.user) -p $(registry.password) -v $(version) -i azureiotedge-diagnostics -n $(namespace) -P azureiotedge-diagnostics
+        displayName: 'Build Image - azureiotedge-diagnostics - amd64'
+      - bash: |
+            scripts/linux/buildEdgelet.sh -i azureiotedge-diagnostics -n microsoft -P iotedge-diagnostics -c $(configuration) --target-arch armv7l &&
+            scripts/linux/buildImage.sh -r $(registry.address) -u $(registry.user) -p $(registry.password) -v $(version) -i azureiotedge-diagnostics -n $(namespace) -P azureiotedge-diagnostics --target-arch armv7l
+        displayName: 'Build Image - azureiotedge-diagnostics - ARM32'
+      - bash: |
+            scripts/linux/buildEdgelet.sh -i azureiotedge-diagnostics -n microsoft -P iotedge-diagnostics -c $(configuration) --target-arch aarch64 &&
+            scripts/linux/buildImage.sh -r $(registry.address) -u $(registry.user) -p $(registry.password) -v $(version) -i azureiotedge-diagnostics -n $(namespace) -P azureiotedge-diagnostics --target-arch aarch64
+        displayName: 'Build Image - azureiotedge-diagnostics - ARM64'
+    variables: 
```

Comment:
Do these variables belong to "job: linux_dotnet_projects"? if so, I think it is better to move up right below "-job".